inc/project.class.php

```diff
@@ -2381,6 +2381,14 @@ static function showKanban($ID) {
                'users_id'  => [
                   'type'         => 'hidden',
                   'value'        => $_SESSION['glpiID']
+               ],
+               'entities_id' => [
+                  'type'   => 'hidden',
+                  'value'  => $_SESSION['glpiactive_entity']
+               ],
+               'is_recursive' => [
+                  'type'   => 'hidden',
+                  'value'  => $_SESSION['glpiactive_entity_recursive']
```

Comment:
I may be wrong, but I think we should not force recursivity by default. Indeed, in GLPI form headers, when creating a new item, the default value is form is `0`.